packages/app/src/vis-packs/core/utils.ts

```diff
@@ -83,3 +89,11 @@ export function getSliceSelection(
 export function getImageInteractions(keepRatio: boolean) {
   return keepRatio ? BASE_INTERACTIONS : INTERACTIONS_WITH_AXIAL_ZOOM;
 }
+
+export function toNumArray(arr: ArrayValue<NumericLikeType>): NumArray {
+  if (typeof arr[0] === 'boolean') {
+    return arr.map((val) => (val ? 1 : 0));
```

Comment:
This apparently performs better than `Number()` on some browsers. `Number()` performs type coercion, which is more complicated than a simple ternary.